Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:8.6
+
+# Create app directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Install app dependencies
+COPY package.json /usr/src/app/
+COPY yarn.lock /usr/src/app/
```

Comment:
could you add a cmd that runs the script?

Reply:
Sure, but I thought better to call with `docker run`. And if you will use docker-compose, there is `command` property in yml file. But if you want to put here, no problem :)